examples/2d/plot_filters.py

```diff
@@ -71,13 +73,14 @@ def colorize(z):
 plt.axis('off')
 plt.set_cmap('gray_r')
 
-f_r = filters_set['phi'][0][..., 0].numpy()
-f_i = filters_set['phi'][0][..., 1].numpy()
-f = f_r + 1j*f_i
+f = filters_set['phi'][0][..., 0]
 
 filter_c = fft2(f)
 filter_c = np.fft.fftshift(filter_c)
-plt.suptitle("The corresponding low-pass filter, also known as scaling function."
-"Color saturation and color hue respectively denote complex magnitude and complex phase", fontsize=13)
+plt.suptitle(("The corresponding low-pass filter, also known as scaling "
+              "function.\nColor saturation and color hue respectively denote "
+              "complex magnitude and complex phase"), fontsize=13)
 filter_c = np.abs(filter_c)
 plt.imshow(filter_c)
+
+plt.show()
```

Comment:
@edouardoyallon Why wasn't this here in the first place?

Reply:
> I added plt.show() so that the plots will stay around for users running the example from the command line rather than an interactive session. Otherwise the figures pop up and immediately disappear when the script reaches the end.

Because I didn't expect an user to run this script in a shell..